Sources/Core/ObjectiveCInitExtension.swift

```diff
@@ -213,7 +213,12 @@ extension ObjCModelRenderer {
 
                 return schemas.map(loop)
             default:
-                return ["\(propertyToAssign) = \(rawObjectName);"]
+                switch schema.memoryAssignmentType() {
```

Comment:
@bkase Any better way to express this switch ... I wanna have a switch expressions :/